edgelet/contrib/config/linux/config.yaml

```diff
@@ -16,11 +16,33 @@
 # Configures the identity provisioning mode of the daemon.
 #
 # Supported modes:
-#     manual   - using an iothub connection string
+#     manual   - using an iothub connection string or a X.509 identity certificate
 #     dps      - using dps for provisioning
 #     external - the device has been provisioned externally.
 #                Uses an external provisioning endpoint to get device specific information.
 #
+# Manual settings when using IoT Hub connection string
+#     device_connection_string - Required. The Edge device connection string
+#                      when using SharedAccessKey authentication mode.
+#                      Ex. "HostName=<hub-name>.azure-devices.net;DeviceId=<device-id>;SharedAccessKey=<key>
+#
+# Manual settings when using a X.509 identity certificate
+#     hub_hostname    - Required. The Azure Iot Hub hostname.
```

Comment:
I think we either need to call this `iothub_hostname` or just `hostname`. The first is more explicit. From a style perspective I think we need to always refer to a hub as an iot hub. The second option is consistent with the connection string.

Reply:
The actual settings calls this "iothub_hostname" and I have fixed the comments section. Thanks for pointing this discrepancy.